packages/istanbul-reports/lib/html/index.js

```diff
@@ -262,7 +262,7 @@ class HtmlReport extends ReportBase {
         this.verbose = opts.verbose;
         this.linkMapper = opts.linkMapper || standardLinkMapper;
         this.subdir = opts.subdir || '';
-        this.date = Date();
+        this.date = new Date();
```

Comment:
These are not equivalent types, `new Date()` returns a Date object, I believe `Date()` returns an ISO string of date.

Do you want `(new Date()).tostring()`?

Reply:
Sure